File: tasks/prac4_js/task9.js
const prompt = require('prompt-sync')();

const a = parseFloat(prompt("Введите сторону A: "));
const b = parseFloat(prompt("Введите сторону B: "));
const c = parseFloat(prompt("Введите сторону C: "));

let triangleType;
// Проверка на существование треугольника
if (a + b <= c || a + c <= b || b + c <= a) {
    triangleType = "не треугольник";
} else if (a === b && b === c) {
    triangleType = "Равносторонний треугольник";
} else if (a === b || a === c || b === c) {
    triangleType = "Равнобедренный треугольник";
} else {
    triangleType = "Разносторонний треугольник";
}

console.log(triangleType);

File: tasks/prac4_python/task9.py
a = int(input("Введите сторону A: "))
b = int(input("Введите сторону B: "))
c = int(input("Введите сторону C: "))

if a + b <= c or a + c <= b or b + c <= a:
    triangle_type = "не треугольник"
elif a == b == c:
    triangle_type = "Равносторонний треугольник"
elif a == b or a == c or b == c:
    triangle_type = "Равнобедренный треугольник"
else:
    triangle_type = "Разносторонний треугольник"

print(triangle_type)
